src/lib/time/__init__.js

```diff
@@ -40,10 +135,16 @@ var $builtinmodule = function (name) {
         return susp;
     });
 
+    mod.sleep.__doc__ = 
+        "sleep(seconds)\n" +
+        "\n" +
+        "Delay execution for a given number of seconds.  The argument may be\n" +
+        "a floating point number for subsecond precision.";
+
     function isLeapYear(year) {
         if((year & 3) != 0) return false;
         return ((year % 100) != 0 || (year % 400) == 0);
-    };
+    }
 
     function getDayOfYear(date) {
```

Comment:
Yep, I just verified it.  I changed the function to:

```
    function getDayOfYear(date,utc) {
        utc = utc || false;
        var dayCount = [0, 31, 59, 90, 120, 151, 181, 212, 243, 273, 304, 334];
        var mn = utc ? date.getUTCMonth() : date.getMonth();
        var dn = utc ? date.getUTCDate() : date.getDate();
        var dayOfYear = dayCount[mn] + dn;
        if(mn > 1 && isLeapYear(utc ? date.getUTCFullYear() : date.getFullYear())) dayOfYear++;
        return dayOfYear;
    };
```
I tried it out on my machine back in Iowa and got the right behavior.